tests/test-spi.js

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2017, Intel Corporation.
+
+console.log("Test SPI APIs");
+console.log("Wire IO11 to IO12");
+
+var spi = require("spi");
+var assert = require("Assert.js");
+
+var spiBus, spiBuffer, bufferData;
+assert.throws(function() {
+    spiBus = spi.open({speed:20, bus:128, polarity:0, phase:0, bits:16});
+}, "SPIBusObject: set SPI bus as '128'");
+
+spiBus = spi.open({speed:20, bus:1, polarity:0, phase:0, bits:16});
+assert(typeof spiBus === "object" && spiBus !== null,
+       "SPIBusObject: be defined");
+
+var spiBusOptions = [
+    [spiBus.bus, "object", "bus", 1],
```

Comment:
Oh I see, @zolkis what do you think here? It does seem like in our constrained environment we should leave out reporting back to the user the settings they used to configure the bus. They should know what they did.

Or we could make it optional for the implementation?